draft-ietf-httpbis-http2.xml

```diff
@@ -2371,6 +2372,11 @@ HTTP2-Settings    = token68
           An endpoint might choose to close a connection without sending GOAWAY for misbehaving
           peers.
         </t>
+        <t>
+          A GOAWAY frame might not immediately precede closing of the connection; a receiver of a

```

Comment:
which might be interpreted (incorrectly) as `MAY not`, which is a common gaffe that @reschke won't allow